tekton/src/tasks/gitops/gitops-delete-kafka-config.yml.j2

```diff
@@ -55,10 +58,18 @@ spec:
         value: $(params.github_pat)
       - name: SM_AWS_REGION
         value: $(params.avp_aws_secret_region)
+      - name: OCP_SERVER
+        value: $(params.cluster_url)
```

Comment:
why we're using `$(params.cluster_url)` to set `OCP_SERVER`?
should this be setting `CLUSTER_URL` or the other way around?

Reply:
value set in param cluster_url is set to env variable OCP_SERVER